### DISCUSSION TITLE
Best way to check for date not in past

### DISCUSSION THREAD

**Discussion by ronvdc** [statamic/cms#4352] [Q&A] [answered]
I'd like to check whether a date is not in the past, so today or in the future. What's the best way to do this?
I know you can do something like
{{ if tour_date | is_future }}

But I can't seem to figure out how to either negate this (so NOT is_past) or combine two conditions (is_today OR is_future).
I don't want to use just is_future, since that leaves out today.

Thanks!
--Ron

**Comment by duncanmcclean** [comment, marked as answer]
Does this work?

```antlers
{{ if {tour_date | is_future} || {tour_date | is_today} }}
```

_I've not tested it so could be completely off but worth a try? 😅_

**Reply by ronvdc** [reply]
I can confirm that this works:

`{{ if tour_date | is_today || tour_date | is_future }}
`

Thanks for pointing me in the right direction!
--Ron
